firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -811,7 +823,7 @@ extension BrowserViewController: WKNavigationDelegate {
             tempPDF.onDownloadError = { error in
                 self?.navigationHandler?.removeDocumentLoading()
                 self?.logger.log("Failed to download Document",
-                                 level: .warning,
+                                 level: .fatal,
```

Comment:
Just saw this log level change, if we send fatal we'll see it in Sentry. This download shouldn't happen too often right? Just to avoid the case where we overwhelm Sentry with fatal logs (happened in the past, with the number of users we have we can rapidly be rate limited).

Reply:
One more question: does it actually help us to track this event in Sentry? Specifically, does it aid in debugging any cases? It basically tells us "some document failed to download", unless you wanted the breadcrumbs?